res/CMakeLists.txt

```diff
@@ -199,20 +199,15 @@ else()
             RENAME      solvespace.svg)
         install(FILES freedesktop/solvespace-scalable.svg
             DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/icons/hicolor/scalable/mimetypes
-            RENAME      application.x-solvespace.svg)
+            RENAME      application-x-solvespace.svg)
```

Comment:
Trying to figure out if this affects flatpak - have you tried building a flatpak with this patch in place? I can't find a mention of mime in the flatpak docs, but I was pretty sure that's why it was named that.

Reply:
Note this branch applies only to non-flatpak and non-snap builds. The flatpak branch has:
```cmake
        install(FILES freedesktop/solvespace-scalable.svg
            DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/icons/hicolor/scalable/apps
            RENAME      com.solvespace.SolveSpace.svg)
        install(FILES freedesktop/solvespace-scalable.svg
            DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/icons/hicolor/scalable/mimetypes
            RENAME      com.solvespace.SolveSpace.svg)
```
also with a custom XDG mime type definition that defines a different `<generic-icon>`. Not sure why that it is needed, instead of relying on the default name of XDG mimetypes (e.g. `application/x-foo` mime -> `application-x-foo` icon).